### PR TITLE
Update feature.yml

### PR DESCRIPTION
**Pull request by kleenpulse**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 1]
```diff
@@ -18,6 +18,27 @@ body:
...
+      description: What are the requirements for this ticket.
+    validations:
+      required: true
+   - type: textarea
+    id: tests
+    attributes:
+      label: Tests
+      description: What are the required Tests that must be passed for your ticket to be considered complete.
+    validations:
+      required: true
```
Comment (ickynavigator) [inline .github/ISSUE_TEMPLATE/feature.yml]: your tests are the basically the same as acceptance criteria

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 2]
```diff
@@ -18,6 +18,27 @@ body:
...
+     - type: textarea
+    id: tasks
+    attributes:
+      label: Tasks
+      description: What are the task(s) that must be completetd for your ticket to be considered complete.
+    validations:
+      required: true
```
Comment (ickynavigator) [inline .github/ISSUE_TEMPLATE/feature.yml:34]: tasks basically overlap with the description of the issue/acceptance criteria

**Review by ickynavigator — CHANGES_REQUESTED** [review]

**Review by kleenpulse — COMMENTED** [review]
Made necessary changes

**Review by kleenpulse — COMMENTED** [review]
fixed

**Pull request closed without merging**